recipes/ecfeed/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.33.0"
+
+class EcFeedConan(ConanFile):
+    name = "ecfeed"
+    version = "1.1.0"
```

Comment:
```suggestion
```

do not declare the version in the recipe, so we can re-use it for any version in the future.

Reply:
Removed.